pkg/ddl/job_scheduler.go

```diff
@@ -764,7 +666,7 @@ func updateDDLJob2Table(
 	if err != nil {
 		return err
 	}
-	sql := fmt.Sprintf(updateDDLJobSQL, util.WrapKey2String(b), job.ID)
+	sql := fmt.Sprintf("update mysql.tidb_ddl_job set job_meta = %s where job_id = %d", util.WrapKey2String(b), job.ID)
```

Comment:
I think original updataDDLJobSQL variable would be better?

Reply:
the reason I inline it is nearly all other place use string literal directly, and it's only used once

I do agree to define it as const if it's or will be used multiple times, but seems we won't